packages/lib-classifier/src/store/TutorialStore.spec.js

```diff
@@ -91,24 +91,36 @@ describe('Model > TutorialStore', function () {
       })
   }
 
+  function setupStores(clientStub, authClientStub) {
+    rootStore = null
```

Comment:
Same comment here as on the UPP tests. Functions should avoid modifying variables outside their own scope. Better to to return the new root store from this function.